pkg/ldbbolt/ldbbolt.go

```diff
@@ -343,7 +343,7 @@ func (bdb *LdbBolt) UpdatePassword(req *ldap.PasswordModifyRequest) error {
 		}
 
 		mod := ldap.ModifyRequest{}
-		mod.DN = ndn
+		mod.DN = req.UserIdentity
```

Comment:
I wonder whether this is enough. I guess ldapentry.ApplyModify is somehow wrong in just doing a simple string compare. I just discoverd that go-ldap provides  dn.EqualFold()  https://github.com/go-ldap/ldap/blob/master/dn.go#L213
Might make sense to switch  https://github.com/libregraph/idm/blob/master/pkg/ldapentry/ldapentry.go#L25 to use that.

Are you willing to look into that?

Reply:
Sure I can do that. :+1: